        0/0 |    STARTUP │ (Re)initialized boot script
        0/0 |   DIRECTOR │ Launched worker 0
        0/1 |      PHASE │ run
        0/1 |      START │ runpy ./plan.py
        1/3 |    SUCCESS │ runpy ./plan.py
        1/3 |      START │ runpy ./work.py case1.txt
        1/2 |       FAIL │ runpy ./work.py case1.txt
────────────────────────────────── Step info ───────────────────────────────────
Command               stepup act runpy ./work.py case1.txt
Return code           0
──────────────────────────────── Invalid inputs ────────────────────────────────
Input changed unexpectedly: cases.txt (digest 125bc6a0 ➜ 2863481f)
────────────────────────────────────────────────────────────────────────────────
        1/2 |      ERROR │ The scheduler has been drained due to unexpected input changes.
        1/2 |    WARNING │ 1 step(s) failed.
        1/2 |    WARNING │ Scheduler is put on hold. Not reporting pending steps.
        1/2 |    WARNING │ Skipping file cleanup due to incomplete build
        1/2 |    WARNING │ Made 1 step(s) in the queue pending.
        1/2 |    WARNING │ Check logs: .stepup/fail.log .stepup/warning.log
        1/2 |      PHASE │ watch
        1/2 |    UPDATED │ cases.txt
        0/3 |      PHASE │ run
        0/1 |      START │ runpy ./plan.py
        1/3 |    SUCCESS │ runpy ./plan.py
        1/3 |      START │ runpy ./work.py case3.txt
        2/3 |    SUCCESS │ runpy ./work.py case3.txt
        2/3 |      START │ runpy ./work.py case4.txt
        3/3 |    SUCCESS │ runpy ./work.py case4.txt
        3/3 |   DIRECTOR │ Trying to delete 1 outdated output(s)
        3/3 |      CLEAN │ case1.txt
        3/3 |      PHASE │ watch
        3/3 |   DIRECTOR │ Stopping workers
        3/3 |   DIRECTOR │ See you!
